### ISSUE TITLE
[RDS] if lambda function is created in a different region from the KMS key => issue

### ISSUE THREAD

**Issue by l0k0ms** [DataDog/dd-aws-lambda-functions#10]
With the Enhanced version of the RDS integration if the lambda function is created in a different region from the KMS key there is an issue. 

It seems our lambda code automatically initializes a kms client for the default region (same region the lambda runs in). 

So the user must create another key in the same region as the lambda function and regenerate the encrypted config.

**Comment by tmichelet** [comment]
Indeed, as a workaround you can probably add the region parameter to this line: https://github.com/DataDog/dd-aws-lambda-functions/blob/master/rds_enhanced_monitoring/main.py#L38

Good feature request for the team. I'd also add the ability to use environment variables (to define the dd api key, kms region, encrypted keys. That way you would never need to edit the function's code
